README.md

```diff
@@ -22,36 +52,26 @@ OPTIONS:
             Print version information
 ```
 
-Check:
+
+## Check the API
+
+Ensure that the explorer backend is functioning correctly by executing the following command:
 
 ```bash
-> curl http://localhost:4000
+> curl http://localhost:4000/api/v1
 Welcome to Iroha 2 Block Explorer!
 ```
 
-## Deploy
-
-- [Install Rust](https://www.rust-lang.org/tools/install)
-- Build binary:
-
-  ```bash
-  cargo build --release
-  ```
+## API
 
-- Prepare Iroha Client config ([reference](https://github.com/hyperledger/iroha/blob/ea45b5053018acd48340024800786ff5a3d0904d/docs/source/references/config.md)). **Define target peer location here**.
+Refer to [Block Explorer API](api.md).
 
-- Run web server:
 
-  ```bash
-  ./target/release/iroha2_explorer_web \
-      -c /path/to/client_config.json \
-      -p 8080 # may be env PORT, default is 4000
-  ```
+## Tools
 
-## API
+The following tools are available in conjunction with the Iroha 2 blockchain explorer :
 
-Refer to [Block Explorer API](api.md).
+ - [genesis-gen](./tools/genesis-gen/README.md): Genesis generator (a tool to generate sample data).
+- [explorer-deploy-dev-tool](https://github.com/0x009922/explorer-deploy-dev-tool): A tool for automating the deployment of Iroha and the explorer.
```

Comment:
I wouldn't include `explorer-deploy-dev-tool` in this list, as I made that one only for convenience of persons I am working with directly.

Reply:
The ``` explorer-deploy-dev-tool``` significantly aided my comprehension of the installation procedure, and the accompanying configs file proved invaluable for swiftly configuring Explorer.
Considering its usefulness, it may be advantageous to include the tool in the main repository, making it accessible to a broader audience.
However, if we decide to exclude it, it would be prudent to include a sample configs file directly in the main repository for ease of reference.